app/templates/releases/index.hbs

```diff
@@ -1,181 +1,100 @@
-<h1>Ember.js Release Cycle</h1>
-
-{{release-timeline project=emberBetaProject}}
+<h1>Ember.js Releases</h1>
 
 <p>
-  Ember releases follow a "train" release model. About every
-  six weeks, the <code>master</code> branch of Ember (sometimes referred to as
-  "canary") branches to become the new beta. After six weeks as a beta, this
-  version becomes the new stable release of Ember.
+  Ember knows how to ship new features without breaking your existing applications.
```

Comment:
This paragraph seems extraneous. Specifically the first sentence.

Reply:
I improved it and made it less extraneous 👍